File: 1794.c
#include <stdio.h>

int main(void) {
   int n, la, lb;

   scanf("%d", &n);
   scanf("%d %d", &la, &lb);

   if (la <= n && n <= lb) {
      scanf("%d %d", &la, &lb);
      if (la <= n && n <= lb)
         printf("possivel");
      else printf("impossivel");
   }
   else {
      scanf("%d %d", &la, &lb);
      printf("impossivel");
   }
   printf("\n");
   return 0;
}

File: outros/1794.py
n = int(input())
e = str(input()).split()
la = int(e[0])
lb = int(e[1])
if la <= n <= lb:
    e = str(input()).split()
    la = int(e[0])
    lb = int(e[1])
    if la <= n <= lb: print('possivel')
    else: print('impossivel')
else:
    input()
    print('impossivel')
